<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calendrier</class>
 <widget class="QDialog" name="calendrier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>321</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:20px;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_affdate">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>170</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_suppD">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>230</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-image: url(:/Downloads/bouton-supprimer.png);
color:white;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_affdate">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius:5px;
   border-right:1px solid #aaaaaa;
	border-bottom:1px solid #aaaaaa;
}
QPushButton:enabled{
background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(190, 233, 231);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton:disabled{
background-color: #aaaaaa;
	color: rgb(255, 255, 255);
}

</string>
   </property>
   <property name="text">
    <string>afficher</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_ajoutdate">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_ajoutdate">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius:5px;
   border-right:1px solid #aaaaaa;
	border-bottom:1px solid #aaaaaa;
}
QPushButton:enabled{
background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(190, 233, 231);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton:disabled{
background-color: #aaaaaa;
	color: rgb(255, 255, 255);
}

</string>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>251</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(190, 233, 231);

alternate-background-color: rgb(98, 187, 203);
color: rgb(27, 75, 101);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
